app/src/main/java/org/mozilla/fenix/library/bookmarks/selectfolder/SelectBookmarkFolderFragment.kt

```diff
@@ -101,11 +99,9 @@ class SelectBookmarkFolderFragment : Fragment(), AccountObserver {
     }
 
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
-        val visitedAddBookmark = SelectBookmarkFolderFragmentArgs.fromBundle(arguments!!).visitedAddBookmark
-        if (!visitedAddBookmark) {
+        val args: SelectBookmarkFolderFragmentArgs by navArgs()
```

Comment:
I'm not familiar with `navArgs` vs `fromBundle(arguments)` are you able to explain why we're making this change?

Reply:
It's just a bit cleaner. The internal code is identical.